### DISCUSSION TITLE
js沙箱会影响vuediff吗

### DISCUSSION THREAD

**Discussion by FisNew** [jd-opensource/micro-app#259] [Q&A]
qiankun坑惨了人，sandbox的实现方式侵入性太强，导致vuediff失效，性能断崖式下跌，不知道micro-app是否规避这个问题？

**Comment by bailicangdu** [comment]
建议写个demo实际测试一下

**Comment by yiludege** [comment]
@FisNew 知道是啥问题导致的性能断崖式下跌吗？我这边测试了一下micro-app，在vue框架下，dom的开销大概是原生的2倍左右，qiankun都快200倍了

**Reply by FisNew** [reply]
qiankun使用proxy做了代理，在比对VNode时，proxy代理返回的VNode和原本的VNode是不等的，但实际上VNode是无变化的。
